src/test/scala/scalaz/nio/ChannelSuite.scala

```diff
@@ -10,7 +10,7 @@ object ChannelSuite extends RTS {
     import harness._
     section(test("read/write") { () =>
       val inetAddress = InetAddress.localHost
-        .flatMap(iAddr => SocketAddress.inetSocketAddress(iAddr, 1337))
+        .flatMap(iAddr => SocketAddress.inetSocketAddress(iAddr, 13370))
```

Comment:
"leeto", why?

Reply:
It conflicted with port occupied by ensime. Will cleanup that later.